index.html

```diff
@@ -1605,6 +1605,108 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+
+		<div class="role" id="code">
+			<rdef>code</rdef>
+			<div class="role-description">
+				<p>An element that represents a fragment of computer code. This includes any string that a computer would recognize.
+				</p>
```

Comment:
We usually try to avoid describing a role as "An element", particularly in the first sentence. Perhaps something like "A section whose content represents..."

Reply:
👍 I'll update the wording here. fwiw I was "copying" the wording for the new [time role](https://pr-preview.s3.amazonaws.com/schne324/aria/pull/969.html#time) which is described as "_an element_"

Reply:
Hmmm, I see that it's used in the new [meter role](http://w3c.github.io/aria/#meter) also, except that for meter it links to a definition for "element" in the ARIA spec, which is:
> **Element**
> In this specification, element is used as it is in markup languages. Elements are the structural elements in markup language that contains the data profile for objects.

I seem to remember a discussion about this a short while ago.
@mcking65, @joanmarie - should we avoid using "An element that..." in the initial definition of a role, or is that ok?